aggregation/navbar.html

```diff
@@ -1,17 +1,17 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/tocbot/4.4.2/tocbot.js"></script>
 
 <script>
-    var oldtoc = document.getElementById('toctitle').nextElementSibling;
-    var newtoc = document.createElement('div');
+    const oldtoc = document.getElementById('toctitle').nextElementSibling;
+    let newtoc = document.createElement('div');
```

Comment:
Why is `newtoc` defined with `let`? As far as I can see this variable is never re-assigned so should be fined to make it a constant.
```suggestion
    const newtoc = document.createElement('div');
```